radixdlt-java/src/main/java/com/radixdlt/client/core/ledger/AtomStore.java

```diff
@@ -48,4 +43,23 @@
 	 * @return a never ending observable of updates
 	 */
 	Observable<AtomObservation> getAtomObservations(RadixAddress address);
+
+	/**
+	 * Retrieve the current set of validated up particles at a given shardable.
+	 * @param address the address to get the particles under
+	 * @return a stream of all up particles of the current local view
```

Comment:
Missing UUID documentation, so I don't know what the intent of this parameter is.

Reply:
Yup, I would agree with this comment.